src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java

```diff
@@ -36,20 +37,24 @@ public static GlobalIgnoreApplier loadRecheckIgnore() {
 		try {
 			final LoadFilterWorker loadFilterWorker = new LoadFilterWorker( NopCounter.getInstance() );
 			return loadFilterWorker.load();
+		} catch ( final FileNotFoundException e ) {
+			logger.error( "Could not find recheck ignore file." );
```

Comment:
We should also log the actual exception (like below):

```suggestion
			logger.error( "Could not find recheck ignore file.", e );
```

Reply:
Actually, when you have a look at the log, (e.g. here https://travis-ci.com/retest/recheck-web/builds/112459315), not logging that exception was one of the motivations for that change. It adds a lot of noice...